docs/developers_guide/micro_frontends_in_open_edx.rst

```diff
@@ -16,35 +16,168 @@ To run an MFE locally you must install `node 12 <https://nodejs.org>`_ (with npm
+  Used by both the build process and application to set the path of the application.  Should include an initial forward slash ``/``.  Will default to ``/`` if not specified.
+
+``BASE_URL``
+  Example: ``https://new-mfe.example.com``
+
+  The fully-qualified URL to the micro-frontend being built. Used by frontend-platform as part of authentication, as well as by many applications for internal links.
+
+Internationalization
+--------------------
+
+``LANGUAGE_PREFERENCE_COOKIE_NAME``
+  Example: ``example-language-preference``
+
+  The name of the cookie where the user's language preferences should be stored.
+
+Service URLs
+------------
+
+``CREDENTIALS_BASE_URL``
+  Example: ``https://credentials.example.com``
+
+  The fully-qualified URL to the credentials service in the target environment.
+
+``DISCOVERY_API_BASE_URL``
+  Example: ``https://discovery.example.com``
+
+  The fully-qualified URL of the `Course Discovery Service <https://github.com/edx/course-discovery>`_ for the target environment.
+
+``PUBLISHER_BASE_URL``
+  Example: ``https://publisher.example.com``
+
+  The fully-qualified URL of the `Publisher Micro-frontend <https://github.com/edx/frontend-app-publisher>`_.  Today this is only used by micro-frontends that are not offically supported in any Open edX release, such as the `Support Tools Micro-frontend <https://github.com/edx/frontend-app-support-tools>`_.  **As such, this should be considered deprecated and will be removed from a future release until such time as publisher itself is included.**
+
+``ECOMMERCE_BASE_URL``
+  ``Example: https://ecommerce.example.com``
+
+  The fully-qualified URL of the `Ecommerce Service <https://github.com/edx/ecommerce>`_ in the target environment.
+
+``LMS_BASE_URL``
+  Example: ``https://courses.example.com``
+
+  The fully-qualified URL to the LMS in the target environment.
+
+``LOGIN_URL``
+  Example: ``https://courses.example.com/login``
+
+  The fully-qualified URL to the login page in the target environment.
+
+``LOGOUT_URL``
+  Example: ``https://courses.example.com/logout``
+
+  The fully-qualified URL to the API endpoint in the target environment which performs a user logout.
+
+``STUDIO_BASE_URL``
+  Example: ``https://studio.example.com``
+
+  The fully-qualified URL of Studio in the target environment. This is often used by micro-frontends from which educators may wish to go edit course content, such as `frontend-app-learning <https://github.com/edx/frontend-app-learning>`_
+
+``MARKETING_SITE_BASE_URL``
+  Example: ``https://www.example.com``
+
+  The fully-qualified URL of the environment's marketing site.  Today this is used by frontend-app-account to link to a demographics collection page, and also as a base URL for optimizely experiment scripts.  **This is deprecated, and will be removed in a future release.** Use cases in frontend-app-account will be refactored to use different environment variables, since neither use case has anything to do with marketing.
```

Comment:
Again I'd pull out the DEPR warning so it's more visible.

I also don't really understand if I need to use this variable or not.

Reply:
Tried to clarify a bit - it's basically required but shouldn't be used for any new code.